source/config/Config.cpp

```diff
@@ -1020,17 +1059,14 @@ bool PlainConfig::PubSub::Validate() const
             DeviceClient::DC_FATAL_ERROR);
         return false;
     }
-    if (publishFile.has_value() && !publishFile->empty())
-    {
-        if (!FileUtils::ValidateFilePermissions(publishFile.value(), Permissions::PUB_SUB_FILES, true))
-        {
-            return false;
-        }
-    }
     if (subscribeFile.has_value() && !subscribeFile->empty())
     {
         if (!FileUtils::ValidateFilePermissions(subscribeFile.value(), Permissions::PUB_SUB_FILES, true))
         {
+            LOGM_ERROR(
+                Config::TAG,
+                "*** %s: Subscribe File field must be specified if Pub-Sub sample feature is enabled ***",
```

Comment:
Why is this being enforced? In the README we state that the pub sub feature does not need to have files specified, and will just publish Hello World and log a message

Reply:
It only enters this if something is specified & it's "bad"